tests/density_estimator_test.py

```diff
@@ -435,3 +438,77 @@ def test_mixed_density_estimator(
     # Test samples
     samples = density_estimator.sample(sample_shape, condition=conditions)
     assert samples.shape == (*sample_shape, batch_dim, *input_event_shape)
+
+
+def test_build_zuko_flow_logit():
+    batch_x = torch.randn(10, 5)
+    batch_y = torch.randn(10, 3)
+    num_dim = 5
+    x_dist = BoxUniform(
+        low=-2 * torch.ones(num_dim), high=2 * torch.ones(num_dim)
+    )  # Define a bounded distribution
+
+    # Test case where x_dist is provided (should not raise an error)
+    try:
+        flow = build_zuko_flow(
+            which_nf="MAF",
+            batch_x=batch_x,
+            batch_y=batch_y,
+            z_score_x="logit",
+            z_score_y="logit",
+            x_dist=x_dist,
+        )
+        assert isinstance(flow, ZukoFlow)
+        print(
```

Comment:
no need for print statement - if the test passes then we know it passes when using `pytest`